kinder/ci/tools/update-workflows/templates/workflows/instance-config-tasks.yaml

```diff
@@ -84,16 +84,16 @@ tasks:
       set -x
 
       docker exec {{ .vars.clusterName }}-control-plane-1 bash -c '
-        nodes=$(KUBECONFIG=/etc/kubernetes/admin.conf kubectl get nodes -o jsonpath='{.items[*].metadata.name}')
+        nodes=$(KUBECONFIG=/etc/kubernetes/admin.conf kubectl get nodes -o jsonpath="{.items[*].metadata.name}")
 
         for node in $nodes; do
-          annotation=$(KUBECONFIG=/etc/kubernetes/admin.conf kubectl get node "$node" -o jsonpath="{.metadata.annotations['kubeadm\.alpha\.kubernetes\.io/cri-socket']}")
+          annotation=$(KUBECONFIG=/etc/kubernetes/admin.conf kubectl get node "$node" -o jsonpath="{.metadata.annotations.kubeadm\.alpha\.kubernetes\.io\/cri-socket}")
```

Comment:
wasn't `['...']` the correct way to handle this?

Reply:
i think both are supposed to work - accessing a key by using `[....`] and by using `.`, no?